### ISSUE TITLE
Task to process cube

### ISSUE THREAD

**Issue by fdw** [Seddryck/NBi#677]
So, maybe we missed something or are doing something weird, but currently, we set up tests first by importing data into the database using `<sql-run>`. Then, we need to explicitly process our cube before we can run queries. To do that, we call a Powershell script with `<exe-run>`.

However, I'm wondering if there might be a better way. I imagine we cannot be the only ones to re-process the cube between tests, so is there something we missed? Or would it be sensible to add a new task just for that (that can call a Powershell script, for example).

**Comment by Seddryck** [comment]
We're reprocessing cubes or part of the cubes with a tiny CLI so we don't need a specific task but the standard  `<exe-run>`.

Nevertheless, I worked once on a task for this but never committed the code. It's definitively possible to implement this. If anyone is willing to contribute, all pull requests are welcome.
